gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -138,52 +138,52 @@ public Consumer<Flux<NotificationRecords>> consumeMinioEvent() {
     }
 
     public void handleMinioEvent(NotificationRecords notificationRecords) {
-        notificationRecords.events().forEach(event -> {
-            LOGGER.debug("s3 event received");
-            switch (event.eventType()) {
-                case OBJECT_CREATED_ANY:
-                case OBJECT_CREATED_PUT:
-                case OBJECT_CREATED_POST:
-                case OBJECT_CREATED_COPY:
-                case OBJECT_CREATED_COMPLETE_MULTIPART_UPLOAD:
-                    updateTasks(event);
-                    break;
-                case OBJECT_REMOVED_ANY:
-                case OBJECT_REMOVED_DELETE:
-                case OBJECT_REMOVED_DELETED_MARKER_CREATED:
-                    removeProcessFile(event);
-                    break;
-                default:
-                    LOGGER.info("S3 event type {} not handled by task manager", event.eventType());
-                    break;
-            }
-        });
+        synchronized (LOCK) {
```

Comment:
Might be preferable to be at lower level (on updateTasks and removeProcessFile) to prevent handling method that works on Flux to lose some messages

Reply:
ok done